Gemfile.lock

```diff
@@ -660,8 +660,8 @@ GEM
     stringex (1.3.3)
     stripe (3.3.1)
       faraday (~> 0.9)
-    therubyracer (0.12.0)
-      libv8 (~> 3.16.14.0)
+    therubyracer (0.12.3)
```

Comment:
One of our staging servers is really old and failed to install mini_racer. We should test deployment on Australian staging1 to make sure this passes (or finally replace that server).